x-pack/test/functional_with_es_ssl/apps/cases/basic/index.ts

```diff
@@ -9,6 +9,7 @@ import { FtrProviderContext } from '../../../ftr_provider_context';
 
 export default ({ loadTestFile }: FtrProviderContext) => {
   describe('Cases basic license', function () {
+    this.tags('skipFIPS');
```

Comment:
@kc13greiner Do you think skipping at this level is acceptable? Given that cases is not expected to be present in a basic license, any tests that get added here would be confirming the feature is not present and thus not be applicable to FIPS.